src/kakarot/instructions/stop_and_math_operations.cairo

```diff
@@ -241,10 +241,11 @@ namespace StopAndMathOperations {
         EXP:
         let range_check_ptr = [ap - 2];
         let popped = cast([ap - 1], Uint256*);
+        tempvar bitwise_ptr = cast([fp - 4], BitwiseBuiltin*);
 
-        let result = uint256_exp(popped[0], popped[1]);
+        let result = uint256_fast_exp(popped[0], popped[1]);
```

Comment:
alternatively we can do like SSJ ->
for popped[1] < 10, slow exp
for popped[1] >= 10, fast exp

but this adds one math cmp